test/src/runner/index.js

```diff
@@ -694,5 +695,16 @@ describe("NodejsEnvRunner", () => {
             assert.notCalled(BrowserRunner.prototype.run);
             assert.notCalled(BrowserRunner.prototype.cancel);
         });
+
+        it.only("should cancel all executing workers", async () => {
```

Comment:
only

Reply:
wow, looks like a commit from my cat. I will remove it!